Realm/RLMObjectSchema.mm

```diff
@@ -276,6 +276,7 @@ - (void)setTable:(realm::Table *)table {
         p.object_type = prop.objectClassName ? prop.objectClassName.UTF8String : "";
         p.is_indexed = prop.indexed;
         p.is_primary = (prop == _primaryKeyProperty);
+        p.table_column = prop.column;
```

Comment:
I don't think this change does anything - everywhere `objectStoreCopy` is used, the column mappings are updated afterwards. This also seems possibly dangerous since we only ever update ObjectSchema instances with column mappings from the table, where here we are copying the mapping in the other direction.